tutorials/btp-cf-buildpacks-java-create/btp-cf-buildpacks-java-create.md

```diff
@@ -0,0 +1,664 @@
+---
+author_name: Gergana Tsakova
+author_profile: https://github.com/Joysie
+title: Create a Java Application via Cloud Foundry Command Line Interface
+description: Create a simple Java application in the Cloud Foundry Command Line Interface (cf CLI) and enable services for it.
+auto_validation: true
+time: 40
+tags: [ tutorial>beginner, software-product>sap-btp--cloud-foundry-environment, software-product-function>sap-btp-cockpit]
+primary_tag: programming-tool>java
+---
+
+
+## Prerequisites
```

Comment:
Please remove all the full stops from the following lines

Reply:
Done!